ldp.js

```diff
@@ -251,8 +251,8 @@ function Ldp (rdf, options) {
   };
 
   var deleteGraph = function (req, res, next, iri, options) {
-    self.graphStore.delete(iri, null, options).then(function () {
-      res.statusCode = 204; // No Content
+    self.graphStore.delete(iri, null, options).then(function (success) {
```

Comment:
We need to check if success is true, otherwise, we should return 404 or is that in the error?